pom.xml

```diff
@@ -52,7 +52,6 @@
             <groupId>info.picocli</groupId>
             <artifactId>picocli-codegen</artifactId>
             <version>${picocli.version}</version>
-            <scope>provided</scope>
```

Comment:
I changed the scope to `compile` (default) because I wanted to use it within the exec plugin.